scripts/init-docker.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
```

Comment:
I suggest adding a GitHub link to BuildTools with commit hash and path to keep track of where this came from.